rundmcmc/make_graph.py

```diff
@@ -1,6 +1,7 @@
-import networkx
+from graph_tool.all import *
```

Comment:
Please don't ever do imports this way.  It makes it very hard to understand where functions and variables come from, and it greatly increases the chances of something strange happening if some function or variable clobbers something from another file.

Did this come from a graph_tool example, or what?

Reply:
The very first code snippet they give 😬
https://graph-tool.skewed.de/static/doc/quickstart.html